docker-for-mac/faqs.md

```diff
@@ -6,28 +6,33 @@ redirect_from:
 title: Frequently asked questions (FAQ)
 ---
 
-## Stable and Edge releases
+## Docker Desktop releases
```

Comment:
I know I wrote this section, but having seen it I'm wondering if it's the right thing to put in the documentation. But I'm open to persuasion in either direction.

What I'm worried about is that it's quite time-limited, and aimed specifically at people who are used to the previous release pattern and are now trying to understand the new one. It made sense in the context of the PRFAQ, but I'm not quite sure in more permanent documentation.

I guess our options are: put it somewhere else; keep it but maybe delete it in a few months; rephrase it somehow; just ignore it and don't include it anywhere.

@usha-mandya @nebuk89 WDYT?

Reply:
I agree, let's keep it for at least a couple of months while we watch the number of users on 'old edge' drain down then we can remove this once we are below say... 5k?